conf/modules.config

```diff
@@ -89,5 +89,14 @@ process {
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
+
+    withName: HAMRONIZATION_DEEPARG {
+        publishDir = [
+            path: { "${params.outdir}/macrel" },
```

Comment:
```suggestion
            path: { "${params.outdir}/deeparg" },
```

Reply:
Fine for now, although I wonder if we even need to export those files at all as it should be summarised in *_SUMMARIZED